catroid/src/org/catrobat/catroid/content/bricks/WhenNfcBrick.java

```diff
@@ -80,6 +80,10 @@ public WhenNfcBrick(WhenNfcScript script) {
 		this.oldSelectedNfcTag = null;
 		this.nfcTag = script.getNfcTag();
 		this.whenNfcScript = script;
+
+		if (script != null && script.isCommentedOut()) {
+			setCommentedOut(true);
+		}
```

Comment:
Why is the clone method not altered in order to set the cloned brick comment state ?